js/src/attribute-mapping/attribute-mapping-source-type-selector.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Subsection from '.~/wcdl/subsection';
+import AppRadioContentControl from '.~/components/app-radio-content-control';
+import AppInputControl from '.~/components/app-input-control';
+import AttributeMappingFieldSourcesControl from '.~/attribute-mapping/attribute-mapping-field-sources-control';
+import AppDocumentationLink from '.~/components/app-documentation-link';
+import HelpPopover from '.~/components/help-popover';
+
+const SOURCE_TYPES = {
+	FIXED: 'fixed',
+	FIELD: 'field',
+};
+
+/**
+ * Renders a selector for choosing the source field.
+ *
+ * @param { Object } props The component props
+ * @param { Array } props.sources The sources available for the selector
+ */
+
+const AttributeMappingSourceTypeSelector = ( { sources = [] } ) => {
+	const [ sourceType, setSourceType ] = useState( SOURCE_TYPES.FIELD );
+
+	return (
+		<>
+			<Subsection.Subtitle className="gla_attribute_mapping_helper-text">
+				{ __(
+					'Choose how to assign a value to the target attribute',
+					'google-listings-and-ads'
+				) }
+			</Subsection.Subtitle>
+			<AppRadioContentControl
+				className="gla-attribute-mapping__radio-control"
+				label={ createInterpolateElement(
+					__(
+						'Use value from existing product field. <help />',
```

Comment:
The `<HelpPopover>` component could be rendered separately and the `createInterpolateElement` wrapping could be omitted from the text.

Reply:
Could you elaborate what do you mean here?

Reply:
The `<help />` renders an icon for toggling a popover and it doesn't have a text to be interpolated so that they could be separated like:

```js
<AppRadioContentControl
  label={
    <>
      { __(
        'Use value from existing product field.',
        'google-listings-and-ads'
      ) }
      <HelpPopover
        id={ `${ SOURCE_TYPES.FIELD }-helper-popover` }
      >
        {
          // ... omitted
```

And the "Set a fixed value" option is the same.